release-notes/8.0/known-issues.md

```diff
@@ -10,4 +10,8 @@ For details about known issues, please refer to the individual repositories:
 
 * [.NET MAUI](https://github.com/dotnet/maui/wiki/Known-Issues/)
 * [Android](https://github.com/xamarin/xamarin-android/wiki/Known-issues-in-.NET)
-* [iOS and macOS](https://github.com/xamarin/xamarin-macios/wiki/Known-issues-in-.NET8)
\ No newline at end of file
+* [iOS and macOS](https://github.com/xamarin/xamarin-macios/wiki/Known-issues-in-.NET8)
+
+## AoT Support in .NET
```

Comment:
```suggestion
## Native AOT Support in .NET
```

Our documentation uses all caps. https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/.

It is probably best to fully qualify "Native AOT" instead of just "AOT", since there are other forms of AOT.